release-notes.md

```diff
@@ -1,6 +1,140 @@
+```
+
+### BREAKING - Interface change "harness.subject"
+
+`harness.subject` is a new interface which allows for the easy manipulation and mocking of the contact that is being "acted on" -- "subject of the test". It is intended to replace logic which previously manipulated the awkward `harness.content.contact` or `harness.state.contacts[0]`.
+
+* The `fillForm()` function simulates completing an app form on the subject's profile page.
+* The `getContactSummary()` function returns the contact summary displayed for the subject.
+* The `getTasks()` function returns the tasks listed on the subject's profile page.
+ 
+ `harness.subject` can be assigned an `object` or a `string`. 
+ 
+Assign `subject` the `_id` together with minified mock contact documents. The harness will hydrate the specified contact.
+ 
+```json
+{
+  "coreVersion": "3.10.3",
+  "subject": "patient_id",
+ 
+  "docs": [
+    {
+      "_id": "patient_id",
+      "name": "Sick Bob",
+      "type": "person",
+      "parent": {
+        "_id": "family_id",
+        "parent": {
+          "_id": "chw_area_id"
+        },
+      }
+    },
+ 
+    {
+      "_id": "family_id",
+      "type": "clinic",
+      "parent": {
+        "_id": "chw_area_id"
+      },
+    },
+  ]
+}
+```
+
+```javascript
+const Harness = require('medic-conf-test-harness');
+const harness = new Harness({ subject: 'chw_id', user: 'supervisor_id' });
+```
+
+Assigning `subject` an `object` skips hydration or any manipulation of the value provided - the exact value set will be the data used. This is powerful and fast, useful for mocking and unit testing - but should likely be avoided for integration testing.
+
+```javascript
+const Harness = require('medic-conf-test-harness');
+const harness = new Harness();
+
+harness.subject = {
+  _id: 'patient_id',
+  name: 'Sick Bob',
+  type: 'person',
+  parent: {
+    _id: 'family_id',
+    type: 'clinic',
+    foo: 'bar',
+  }
+};
+
+```
+
+Scenario | Before | After
+-- | -- | --
+Get DoB of the contact I'm testing | `harness.content.contact.date_of_birth` | `harness.subject.date_of_birth`
+Make this test for a family | `harness.state.contacts[0] = { _id: 'family', type: 'clinic' }` | `harness.subject = 'family_id';`
+What tasks are on the CHWs profile | `harness.state.contacts.push(harness.user.parent.contact);` | `harness.subject = harness.user;`
+Only show tasks for contact I'm testing | `harness.state.contacts = [harness.content.contact];` | default behaviour
+
+### Breaking Changes
+1. Tests are now powered by the RulesEngine v2 which was released in cht-core v3.8 which included numerous [breaking changes](https://github.com/medic/cht-core/blob/master/release-notes/docs/3.8.0.md#breaking-changes)
+    1. Projects running cht-core <3.8 should continue to use medic-conf-test-harness@1.x.
+    2. Typically, if a test breaks because of this changes - that’s a good thing. You have found a bug in the production behaviour of your application.
```

Comment:
```suggestion
    2. Typically, if a test breaks because of these changes - that’s a good thing. You have found a bug in the production behaviour of your application.
```